pkg/backend/display/options.go

```diff
@@ -57,6 +57,7 @@ type Options struct {
 	Stderr                 io.Writer           // the writer to use for stderr. Defaults to os.Stderr if unset.
 	SuppressTimings        bool                // true to suppress displaying timings of resource actions
 	SuppressProgress       bool                // true to suppress displaying progress spinner.
+	SuppressLinkToCopilot  bool                // true to suppress displaying a 'explainFailure' link to Copilot.
```

Comment:
Should this be `ShowLinkToCopilot` to default it to false?

And should it then only be conditionally set to true when the user is using the Pulumi Cloud (api.pulumi.com) backend?

Reply:
`SuppressPermalink` will be set to `false` for DIY backend, this is the flag that's checked. 
We decided that the feature should be on by default, hence 'Suppress...' and not 'Show...' in the name of the flag.